request.go

```diff
@@ -117,6 +117,30 @@ func (r *Request) SetHeaders(headers map[string]string) *Request {
 	return r
 }
 
+// SetMultiValueHeaders sets multiple headers fields and its values is list of strings at one go in the current request.
+//
+// For Example: To set `Accept` as `text/html, application/xhtml+xml, application/xml;q=0.9, image/webp, */*;q=0.8`
+//
+// 		client.R().
+//			SetMultiValueHeaders(map[string][]string{
+//				"Accept": []string{"text/html", "application/xhtml+xml", "application/xml;q=0.9", "image/webp", "*/*;q=0.8"},
+//			})
+// Also you can override header value, which was set at client instance level.
+func (r *Request) SetMultiValueHeaders(headers map[string][]string) *Request {
+	for key, values := range headers {
+		var headerValue string
+
+		if len(values) > 1 {
+			headerValue = strings.Join(values, ", ")
+		} else {
+			headerValue = values[0]
+		}
+
+		r.SetHeader(key, headerValue)
```

Comment:
Actually, looking at this again, can the whole loop be replaced with the following?

```go
for key, values := range headers {
    r.SetHeader(key, strings.Join(values, ", "))
}
```

Reply:
Yes, it'll make a cleaner code.